Source/monstdat.cpp

```diff
@@ -335,6 +201,450 @@ const _monster_id MonstConvTbl[] = {
 	MT_LRDSAYTR,
 };
 
+// P_("monster", "A \"monster\"") => "A \"monster\""
+tl::expected<std::string, std::string> ParsePgettext(std::string_view value)
+{
+	constexpr std::string_view Prefix = "P_(\"";
+	constexpr std::string_view Suffix = "\")";
+	if (!value.starts_with(Prefix) || !value.ends_with(Suffix)) {
+		return tl::make_unexpected("Not a P_-string");

```

Comment:
I think it would be better to simply return the string here, the monster name simply isn't translatable at that point. But that is also allowed now.